molecule/command/init/scenario.py

```diff
@@ -159,8 +178,15 @@ def _default_scenario_exists(ctx, param, value):  # pragma: no cover
     type=click.Choice(config.molecule_verifiers()),
     default='testinfra',
     help='Name of verifier to initialize. (testinfra)')
+@click.option(
+    '--driver-template',
+    type=click.Path(
+        exists=True, dir_okay=True, readable=True, resolve_path=True),
+    help="Path to a cookiecutter custom template to initialize the scenario. "
```

Comment:
Should refer more specifically to the driver template. With this description, it is not clear that it does anything new from the existing scenario template overriding.

Reply:
How about this:

```
Path to a cookiecutter custom driver template to initialize the scenario. If the driver template is not found locally, default template will be used instead.
```

Reply:
Good!